sdk/core/azure-core/src/main/java/com/azure/core/util/polling/OperationResourcePollingStrategy.java

```diff
@@ -99,19 +101,41 @@ public OperationResourcePollingStrategy(HttpPipeline httpPipeline, String endpoi
         String operationLocationHeaderName, Context context) {
         this(httpPipeline, endpoint, serializer,
             operationLocationHeaderName == null ? null : HttpHeaderName.fromString(operationLocationHeaderName),
-            context);
+            null, context);
     }
 
     private OperationResourcePollingStrategy(HttpPipeline httpPipeline, String endpoint,
-        ObjectSerializer serializer, HttpHeaderName operationLocationHeaderName, Context context) {
+        ObjectSerializer serializer, HttpHeaderName operationLocationHeaderName, String serviceVersion, Context context) {
         this.httpPipeline = Objects.requireNonNull(httpPipeline, "'httpPipeline' cannot be null");
         this.endpoint = endpoint;
         this.serializer = serializer != null ? serializer : new DefaultJsonSerializer();
         this.operationLocationHeaderName = (operationLocationHeaderName == null)
             ? DEFAULT_OPERATION_LOCATION_HEADER : operationLocationHeaderName;
+
+        this.serviceVersion = serviceVersion;
         this.context = context == null ? Context.NONE : context;
     }
 
+    /**
+     * Creates an instance of the operation resource polling strategy.
+     *
+     * @param httpPipeline an instance of {@link HttpPipeline} to send requests with.
+     * @param endpoint an endpoint for creating an absolute path when the path itself is relative.
+     * @param serializer a custom serializer for serializing and deserializing polling responses.
+     * @param operationLocationHeaderName a custom header for polling the long-running operation.
+     * @param serviceVersion the service version that will be added as query param to each polling
+     * request and final result request URL. If the request URL already contains a service version, it will be replaced
+     * by the service version set in this constructor.
+     * @param context an instance of {@link com.azure.core.util.Context}.
+     */
+    public OperationResourcePollingStrategy(HttpPipeline httpPipeline, String endpoint, ObjectSerializer serializer,
+                                            String operationLocationHeaderName, String serviceVersion, Context context) {
```

Comment:
Could we introduce this constructor overload using `HttpHeaderName` instead of `String` for `operationLocationHeaderName`?